tests/MongoDB.Driver.Tests/Specifications/retryable-writes/TestRunner.cs

```diff
@@ -42,8 +46,8 @@ public void RunTestDefinition(TestCase testCase)
             VerifyServerRequirements(definition);
             VerifyFields(definition, "path", "data", "minServerVersion", "maxServerVersion", "tests");
 
-            var collection = InitializeCollection(definition);
-            RunTest(collection, test, async);
+            InitializeCollection(definition);
```

Comment:
Postpone creating the collection object used for the test itself until later, after the client is created.

The client is created later now also because the JSON files now have clientOptions for each test.